ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Transformer;
+
+use OpenOrchestra\ApiBundle\Exceptions\HttpException\RoleNotFoundHttpException;
+use OpenOrchestra\ApiBundle\Exceptions\TransformerParameterTypeException;
+use OpenOrchestra\ApiBundle\Facade\NodeGroupRoleFacade;
+use OpenOrchestra\Backoffice\Collector\RoleCollector;
+use OpenOrchestra\BackofficeBundle\Model\NodeGroupRoleInterface;
+use OpenOrchestra\BaseApi\Facade\FacadeInterface;
+use OpenOrchestra\BaseApi\Transformer\AbstractTransformer;
+
+/**
+ * Class NodeGroupRoleTransformer
+ */
+class NodeGroupRoleTransformer extends AbstractTransformer
+{
+    protected $nodeRoleGroupClass;
+    protected $collector;
+
+    /**
+     * @param string        $nodeRoleGroupClass
+     * @param RoleCollector $collector
+     */
+    public function __construct($nodeRoleGroupClass, RoleCollector $collector)
```

Comment:
You need in the method reverseTransform of instance nodeGroupRoleInterface to nodeRoleGroupClass. Why not indicated in the settings of the construct ?
---
if (!$source instanceof NodeGroupRoleInterface) {
      $source = new $this->nodeRoleGroupClass();
}
--